frontend/src/page/Realm.tsx

```diff
@@ -35,32 +41,47 @@ export const Realm: React.FC<Props> = ({ path }) => {
         return <b>{"Realm path not found :("}</b>;
     }
 
+    if (!realm.parent) {
+        return unreachable("non root realm has no parent");
+    }
+
     // Prepare data for breadcrumbs
-    const breadcrumbs = realm.parents.slice(1).map(({ name, path }) => ({
-        label: name,
-        href: `/r${path}`,
-    }));
+    const breadcrumbs = realm.parents
+        .slice(1)
+        .concat(realm)
+        .map(({ name, path }) => ({
+            label: name,
+            link: `/r${path}`,
+        }));
```

Comment:
This is probably overkill, but a closure that does (a variation) of this is created thrice in this file alone. Should this be a utility function somewhere? :thinking:

Reply:
Nah, I'd say we rather not.

Reply:
Ok sure :shrug: